halyard-config/src/main/java/com/netflix/spinnaker/halyard/config/model/v1/plugins/Plugin.java

```diff
@@ -37,18 +39,24 @@
   public Boolean enabled;
   public String manifestLocation;
 
+  @Getter(AccessLevel.NONE)
+  public Manifest manifest;
+
   @Override
   public String getNodeName() {
     return name;
   }
 
-  public Manifest generateManifest() {
+  public Manifest getManifest() {
```

Comment:
It seems dangerous that we're caching the result of `getManifest` on the object when the object isn't immutable.  What if `manifestLocation` is updated, then we call `getManifest()`---we'll get an incorrect result.  Can the properties that determine the manifest be made immutable?